pkg/ssh/suftp.go

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright 2023 ghostloda
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ssh
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/pkg/sftp"
+	"golang.org/x/crypto/ssh"
+)
+
+const promptpwd string = "gimmeyourpwdnow"
+const promptsuccess string = "thesudopwdwasok"
+const prompterror string = "ohmgsudofailed!"
+const promptlength int = 15
+
+func NewSudoSftpClient(conn *ssh.Client, sudopwd string, opts ...sftp.ClientOption) (*sftp.Client, error) {
```

Comment:
check serverpaths is exsits?

Reply:
replaced with **grep serverpaths  from '/etc/ssh/sshd_config'** , if file not found return error.